ModulusChecking/Loaders/SortCodeSubstitution.cs

```diff
@@ -8,13 +8,11 @@ namespace ModulusChecking.Loaders
 {
     internal class SortCodeSubstitution
     {
-        private Dictionary<string, string> _sortCodeSubstitutionSource;
+        private static Dictionary<string, string> _sortCodeSubstitutionSource;
 
-        private void SetupDictionary()
+        public SortCodeSubstitution(string scsubtabFileContents)
         {
-            if (_sortCodeSubstitutionSource != null) return;
-
-            _sortCodeSubstitutionSource = Resources.scsubtab
+            _sortCodeSubstitutionSource = scsubtabFileContents
```

Comment:
Maybe simple null checking here?

Reply:
good spot! potentially user provided content now and we're giving poor feedback if it's obviously wrong